iOS/Article/ArticleViewController.swift

```diff
@@ -233,6 +231,61 @@ class ArticleViewController: UIViewController, MainControllerIdentifiable {
 		return currentWebViewController?.webView?.scrollView
 	}
 	
+	@objc
+	func configureAppearanceMenu(_ sender: Any? = nil) {
+		
+		var themeActions = [UIAction]()
+		
+		for themeName in ArticleThemesManager.shared.themeNames {
+			let action = UIAction(title: themeName,
+								  image: nil,
+								  identifier: nil,
+								  discoverabilityTitle: nil,
+								  attributes: [],
+								  state: ArticleThemesManager.shared.currentThemeName == themeName ? .on : .off,
+								  handler: { action in
+				ArticleThemesManager.shared.currentThemeName = themeName
+			})
+			themeActions.append(action)
+		}
+		
+		let defaultThemeAction = UIAction(title: NSLocalizedString("Default", comment: "Default"), image: nil, identifier: nil, discoverabilityTitle: nil, attributes: [], state: ArticleThemesManager.shared.currentThemeName == AppDefaults.defaultThemeName ? .on : .off) { _ in
+			ArticleThemesManager.shared.currentThemeName = AppDefaults.defaultThemeName
+		}
+		themeActions.append(defaultThemeAction)
+		
+		let themeMenu = UIMenu(title: "Theme", image: AppAssets.themeImage, identifier: nil, options: .singleSelection, children: themeActions)
+		themeMenu.subtitle = NSLocalizedString("Change the look of articles.", comment: "Change theme")
+		
+		var children: [UIMenuElement] = [themeMenu]
+		
+		if let currentWebViewController = currentWebViewController {
+			if currentWebViewController.isFullScreenAvailable {
+				let fullScreenAction = UIAction(title: NSLocalizedString("Full Screen", comment: "Full Screen"),
+												image: UIImage(systemName: "arrow.up.backward.and.arrow.down.forward"),
```

Comment:
I went ahead and merged this, but could we get this image moved to AppAssets please?